NAMESPACE

```diff
@@ -163,6 +163,7 @@ export(setDefaultAggregationFunctions)
 export(setDefaultAggregationLabels)
 export(setDefaultErrorbarCapSize)
 export(setDefaultExportParameters)
+export(setDefaultLLOQLinetype)

```

Comment:
This function has been added in a previous PR and the documentation was not updated, right?

Reply:
Yes